packages/recheck/src/main.ts

```diff
@@ -28,6 +28,19 @@ export async function check(
   flags: string,
   params: Parameters & HasAbortSignal = {},
 ): Promise<Diagnostics> {
+  /* c8 ignore stop */
+  if (typeof source !== "string") {
+    throw new TypeError(
+      `Expected 'source' to be a string, but got ${typeof source}`,
+    );
+  }
+  /* c8 ignore stop */
+  if (typeof flags !== "string") {
+    throw new TypeError(
+      `Expected 'flags' to be a string, but got ${typeof flags}`,
+    );
+  }
```

Comment:
It does not make sense. `c8 ignore stop` should be used with `c8 ignore start`. (See the [reference](https://github.com/bcoe/c8/blob/main/README.md#ignoring-uncovered-lines-functions-and-blocks).)

```suggestion
  /* c8 ignore start */
  if (typeof source !== "string") {
    throw new TypeError(
      `Expected 'source' to be a string, but got ${typeof source}`,
    );
  }
  if (typeof flags !== "string") {
    throw new TypeError(
      `Expected 'flags' to be a string, but got ${typeof flags}`,
    );
  }
  /* c8 ignore stop */
```